docs/dev/analysis/features/cht-axis-title.rst

```diff
@@ -0,0 +1,231 @@
+
+Axis Title
+==========
+
+A chart axis has a title which contains a text_frame that you can write
+text to. However, before you can add a title, you must set has_title
+to True.
+
+
+MS API Protocol
+---------------
```

Comment:
I would swap the order of the MS API protocol to come after the proposed protocol. I know that's a little unnatural as far as derivation order, but it turns out better as we get further along. Basically the protocol demonstration is the heart of the enhancement proposal, the rest is all supporting background :)